taichi/ir/scratch_pad.h

```diff
@@ -224,8 +234,6 @@ class ScratchPads {
   void print() {
     for (auto &it : pads) {
       TI_P(it.first->node_type_name);
-      TI_P(it.second.bounds[0]);
-      TI_P(it.second.bounds[1]);
```

Comment:
nit: If this function is merely used for debugging, why not printing bounds as well?

Reply:
Fixed :)